aten/src/ATen/native/cuda/LinearAlgebraStubs.cpp

```diff
@@ -0,0 +1,184 @@
+#include <ATen/Context.h>
+#include <ATen/cuda/CUDAContext.h>
+#include <ATen/cuda/CUDAConfig.h>
+#include <ATen/NativeFunctions.h>
+#include <ATen/Dispatch.h>
+#include <ATen/DynamicLibrary.h>
+#include <ATen/NativeFunctions.h>
+#include <ATen/native/cuda/MiscUtils.h>
+#include <ATen/native/Resize.h>
+#include <ATen/native/LinearAlgebra.h>
+#include <ATen/native/BatchLinearAlgebra.h>
+#if AT_MAGMA_ENABLED()
+#include <ATen/cuda/detail/CUDAHooks.h>
+#include <iostream>
+
+#if defined(BUILD_LAZY_CUDA_LINALG)
+namespace {
+struct MagmaInitializer {
+  MagmaInitializer() {
+    ::at::cuda::detail::set_magma_init_fn([]{ });
+  };
+} initializer;
+}  // namespace (anonymous)
+#endif
+
+#endif
+namespace at {
+namespace native {
+#if !defined(BUILD_LAZY_CUDA_LINALG)
+std::tuple<Tensor, Tensor> legacy_lstsq_cuda_impl(const Tensor &B, const Tensor &A);
+std::tuple<Tensor, Tensor> _solve_helper_cuda_impl(const Tensor& self, const Tensor& A);
+std::tuple<Tensor, Tensor> _symeig_helper_cuda_impl(const Tensor& self, bool eigenvectors, bool upper);
+Tensor _cholesky_solve_helper_cuda_impl(const Tensor& self, const Tensor& A, bool upper);
+std::tuple<Tensor, Tensor> _linalg_qr_helper_cuda_impl(const Tensor& input, c10::string_view mode);
+Tensor& _linalg_inv_out_helper_cuda_impl(Tensor &result, Tensor& infos_lu, Tensor& infos_getri);
+#else
+namespace {
+std::tuple<Tensor, Tensor> (*legacy_lstsq_cuda_impl)(const Tensor &B, const Tensor &A) = nullptr;
+std::tuple<Tensor, Tensor> (*_solve_helper_cuda_impl)(const Tensor& self, const Tensor& A) = nullptr;
+std::tuple<Tensor, Tensor> (*_symeig_helper_cuda_impl)(const Tensor& self, bool eigenvectors, bool upper) = nullptr;
+Tensor (*_cholesky_solve_helper_cuda_impl)(const Tensor& self, const Tensor& A, bool upper) = nullptr;
+std::tuple<Tensor, Tensor> (*_linalg_qr_helper_cuda_impl)(const Tensor& input, c10::string_view mode) = nullptr;
+Tensor& (*_linalg_inv_out_helper_cuda_impl)(Tensor &result, Tensor& infos_lu, Tensor& infos_getri) = nullptr;
+}
+
+at::DynamicLibrary& getTorchLinalgLibrary() {
+  static at::DynamicLibrary lib("libtorch_cuda_linalg.so");
+  return lib;
+}
+
+Tensor& _linalg_inv_out_helper_cuda(Tensor &result, Tensor& infos_lu, Tensor& infos_getri) {
+    if (!_linalg_inv_out_helper_cuda_impl) {
+          _linalg_inv_out_helper_cuda_impl = reinterpret_cast<decltype(&_linalg_inv_out_helper_cuda)>(getTorchLinalgLibrary().sym("_ZN2at6native32_linalg_inv_out_helper_cuda_implERNS_6TensorES2_S2_"));
+          TORCH_CHECK(_linalg_inv_out_helper_cuda_impl != nullptr, "Can't find");
+        }
+    return _linalg_inv_out_helper_cuda_impl(result, infos_lu, infos_getri);
+}
```

Comment:
Why is this one different to all the others?

Reply:
Sorry, that was an accidental leftover from the previous iteration. Rectified.